build.sbt

```diff
@@ -569,8 +569,10 @@ lazy val library = configureAsSubproject(project)
   .settings(
     name := "scala-library",
     description := "Scala Standard Library",
-    Compile / scalacOptions ++= Seq[String]("-sourcepath", (Compile / scalaSource).value.toString),
-    Compile / scalacOptions += "-Xlint:-deprecation,-inaccessible,-nonlocal-return,-valpattern,-doc-detached,_",
+    Compile / scalacOptions ++= Seq("-sourcepath", (Compile / scalaSource).value.toString),
+    Compile / scalacOptions ++= Seq("-feature", "-Xlint", "-Werror"),
+    // disallow old deprecateds. we should keep clamping down further
+    Compile / scalacOptions ++= Seq("-Wconf:cat=deprecation:is", "-Wconf:cat=deprecation&since<2.13:e"),
```

Comment:
`cat=FeatureImplicitConversions:e`

Reply:
yeah, bit of a mix of old (`-feature`) and new (`-Wconf:...`) styles here. I figured people already know at a glance what `-feature` means at a glance, but whatevs, either way